python/ray/tests/test_output.py

```diff
@@ -527,6 +527,21 @@ def __repr__(self):
     assert actor_repr not in out
 
 
+def test_node_name_in_raylet_death():
+    NODE_NAME = "RAY_TEST_RAYLET_DEATH_NODE_NAME"
+    script = f"""
+import ray
+import time
+
+ray.init(_node_name=\"{NODE_NAME}\")
+# This will kill raylet without letting it exit gracefully.
+ray.worker._global_node.kill_raylet()
+time.sleep(30 + 5)
```

Comment:
Instead of sleeping 35 seconds, can you reduce the timeout? You can do it by modifying these system config;

```
/// The duration between heartbeats sent by the raylets.
RAY_CONFIG(uint64_t, raylet_heartbeat_period_milliseconds, 1000)
/// If a component has not sent a heartbeat in the last num_heartbeats_timeout
/// heartbeat intervals, the raylet monitor process will report
/// it as dead to the db_client table.
RAY_CONFIG(int64_t, num_heartbeats_timeout, 30)
```

Reply:
Also, this might not pass if you use WARNING instead of ERROR

Reply:
Ok! So I guess its just env vars...